### DISCUSSION TITLE
Why FAT32 and not exFAT?

### DISCUSSION THREAD

**Discussion by ghost** [OnionUI/Onion#1097] [General]
I just wonder why only support FAT32 and not exFAT? The Miyoo Mini supports exFAT.

I also wonder why Microsoft limits FAT32 to 32GB drives? I think it's because to promote NTFS and exFAT. Because FAT32 supports 2TB (at 512 sector size). Or is there a technical reason. Because Windows does not detect the SD card anymore. I installed Onion OS. But after plugged in again in PC to transfer files, the drive is not detected anymore.

Update: I tried the SD card on a Linux OS (Steam Deck) and it works. So I guess Windows is either blocking FAT32 drives with > 32 GB (64 GB) or just have troubles. ...

**Comment by thomasklemenc** [comment]
1) the Miyoo Mini does not support exFAT

2) Windows has no problem with FAT32 SD card bigger than 32GB, it just doesn't let you format them to FAT32

**Comment by ghost** [comment]
Ok, I guess I was wrong. I thought I saw exFAT when I first plugged the Stock SD card into the PC. But I tried it again with exFAT and it doesn't work. The SD is not mount.

In my case it seems that Windows 11 has problems with FAT32 on this SD card. I'll replace this card with a known brand soon. Maybe I selected the wrong cluster size. First I formatted the 64 GB with a cluster size of 512 bytes. Now I formatted with the more common 4 KB and it seems to work better on Windows. With 512 (which is recommended in the [guide](https://github.com/OnionUI/Onion/wiki/fat32format), I had reading errors on Windows 11. But it also shows a message on plug to repair the drive and repair does not work. Maybe just another useless dialog in Windows. And maybe issues of the stock SD card, which is not recommended. My new brand card is coming soon. I hope it will work better....
